pkg/scaffold/cmd.go

```diff
@@ -74,40 +73,38 @@ func main() {
 
 	namespace, err := k8sutil.GetWatchNamespace()
 	if err != nil {
-		log.Fatal("Failed to get watch namespace: %v", err)
+		logrus.Fatalf("Failed to get watch namespace: %v", err)
```

Comment:
do we want to introduce `logrus` as the default logging dependency? I am not in favor of choosing a logging library for the user. I think we should just use the `log` library provided by go to be neutral. it is up to user to figure out what kinda of logging mechanism they that want.

Reply:
This was done more to make the logging output for the example consistent.
Since we already used logrus in our generated main.go
https://github.com/operator-framework/operator-sdk/blob/refactor/controller-runtime/pkg/scaffold/cmd.go#L66-L68

But I'm going to remove the use of logrus from all the templates now. We can decide later if we want to use it by default in the generated examples.